cli/src/pcluster/constants.py

```diff
@@ -66,13 +66,16 @@
 LUSTRE = "LUSTRE"
 OPENZFS = "OPENZFS"
 ONTAP = "ONTAP"
+FILECACHE = "FILECACHE"
```

Comment:
*Observation to address in a separate PR, but worth starting the discussion for it.*

I understand that we need to add the constant `FILECACHE` and `FSX_FILE_CACHE` because we have always done this way for other storage types.

Nevertheless, we should improve this by requiring just one constant.

Reply:
we have 2 different constants for each to improve readability of the code. I would have to use to_pascal_case() or to_snake_case(),etc at multiple places in the code.